cython_setuptools/vendor.py

```diff
@@ -14,11 +14,11 @@
 import configparser
 
 
-DEFAULTS_SECTION = 'cython-defaults'
-MODULE_SECTION_PREFIX = 'cython-module:'
-CYTHON_EXT = '.pyx'
-C_EXT = '.c'
-CPP_EXT = '.cpp'
+DEFAULTS_SECTION = "cython-defaults"
```

Comment:
You probably need to introduce the flake8/black config before this commit.
I think this change uses a custom config.

Reply:
I'll move the aesthetics just at the end